eam-light-backendejb/src/main/java/ch/cern/cmms/eamlightejb/index/IndexGrids.java

```diff
@@ -50,12 +61,18 @@ private List<IndexResult> searchEquipment(InforContext inforContext,String keywo
             map.put("organization", "organization");
             GridRequest gridRequest = new GridRequest(gridName);
             gridRequest.setUserFunctionName(gridName);
-            gridRequest.addFilter("equipmentno", keyword, operator, GridRequestFilter.JOINER.OR);
 
             if (searchExtraColumns) {
-                gridRequest.addFilter("alias", keyword, operator, GridRequestFilter.JOINER.OR);
+                gridRequest.addFilter("alias", keyword, operator, GridRequestFilter.JOINER.OR, true, false);
                 gridRequest.addFilter("serialnumber", keyword, operator, GridRequestFilter.JOINER.OR);
             }
+            gridRequest.addFilter("equipmentno", keyword, operator, GridRequestFilter.JOINER.AND, false, true);
```

Comment:
If searchExtraColumns is false, aren't we adding a right parenthesis without adding a left one?